glide.lock

```diff
@@ -65,7 +67,7 @@ imports:
   - context
   - publicsuffix
 - name: golang.org/x/sys
-  version: 7a6e5648d140666db5d920909e082ca00a87ba2c
+  version: 99f16d856c9836c42d24e7ab64ea72916925fa97
   subpackages:
   - unix
 testImports: []
```

Comment:
Some notes from the [readme](https://github.com/Masterminds/glide)

A glide.lock file will be created or updated with the dependencies pinned to specific versions. For example, if in the glide.yaml file a version was specified as a range (e.g., ^1.2.3) it will be set to a specific commit id in the glide.lock file. That allows for reproducible installs.

Glide install will read the glide.lock file and install the commit id specific versions there.